config/routes.rb

```diff
@@ -10,7 +10,11 @@
   resources :movies
   resources :halls
   resources :reservations, excep: %i[show new create]
```

Comment:
there is a typo :) `except` instead of `excep`

Reply:
corrected and will be included in next pr, thanks :D